packages/tic-tac-toe-bot/.env-cmdrc

```diff
@@ -0,0 +1,38 @@
+{
+  "development": {
+    "BASE_URL": "http://localhost:9000",
+    "ETH_ADDRESS": "0x1BdF54355A98b43951dB6f5369Dd1bAe31bF2FB0",
+    "PRIVATE_KEY": "0x931D29EE9600C15AE40F14BFA27E82F840C3A8375E21306E34894A9999E632B7",
+    "TIER": "development",
+    "STORE_PREFIX": "HR-bot-development",
+    "DB_ENGINE": "pg",
+    "DB_CONNECTION_STRING": "postgresql://postgres@localhost:5432/postgres",
```

Comment:
This DB string is the same target as the `playground-server`'s DB. These should be distinct DBs.

Also why is there a DB at all?